.github/DISCUSSION_TEMPLATE/general.yml

```diff
@@ -0,0 +1,70 @@
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for contributing to the discussion! Please fill out this form as completely as possible to help us support you better.
+
+  - type: input
+    id: description
+    attributes:
+      label: What happened?
+      description: Clearly describe the problem or question you have.
+      placeholder: e.g., "When I try to launch the tool, I receive an error..."
+    validations:
+      required: true
+
+  - type: textarea
+    id: expected_behavior
+    attributes:
+      label: Expected Behavior (optional)
+      description: What did you expect to happen?
+      placeholder: e.g., "The application should start and show a dashboard..."
+
+  - type: textarea
+    id: steps_to_reproduce
+    attributes:
+      label: Steps to Reproduce
+      description: Describe the exact steps to reproduce the issue.
+      placeholder: |
+        1. Go to '...'
+        2. Click on '...'
+        3. Start the application
+        4. Observe the issue
+    validations:
+      required: true
+
+  - type: textarea
+    id: context_info
+    attributes:
+      label: Context Information
+      description: Provide any relevant context like configuration files (remove sensitive information).
+      placeholder: |
+        {
+          "setting1": "value",
+          "setting2": "value"
+        }
+
+  - type: textarea
+    id: log_output
+    attributes:
+      label: Relevant log output
+      description: Share relevant logs or error messages.
+      placeholder: |
+        [Error] Failed to start the application: Connection refused
```

Comment:
Hint that we expect not only a single line. We expect ideally the full log without sensitive information.